src/calculateRentalCost.js

```diff
@@ -28,7 +28,8 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  // eslint-disable-next-line
+  return (days >= 7) ? days * 40 - 50 : (days >= 3) ? days * 40 - 20 : days * 40;
```

Comment:
I think you need more variables for more understanding what is all this numbers and why we, in one moment, substraction them, and, in other moment, multiply them.